ratis-common/src/main/java/org/apache/ratis/RaftConfigKeys.java

```diff
@@ -61,6 +63,16 @@ static SupportedDataStreamType type(RaftProperties properties, Consumer<String>
     static void setType(RaftProperties properties, SupportedDataStreamType type) {
       set(properties::set, TYPE_KEY, type.name());
     }
+
+    String ENABLED_KEY = PREFIX + ".enabled";
```

Comment:
Instead of add a new configuration, we may add a new type SupportedDataStreamType.DISABLED.  Then, we do not have to complicate the code.

Reply:
@szetszwo updated to using `SupportedDataStreamType.DISABLED`. Can you take a look?